modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/RaftGroupServiceImpl.java

```diff
@@ -603,7 +604,17 @@ else if (resp0.errorCode() == RaftError.EBUSY.getNumber() ||
                         resp0.errorCode() == (RaftError.EAGAIN.getNumber()) ||
                         resp0.errorCode() == (RaftError.ENOENT.getNumber())) { // Possibly a node has not been started.
                         executor.schedule(() -> {
-                            sendWithRetry(peer, req, stopTime, fut);
+                            Peer targetPeer = peer;
+
+                            if (resp0.errorCode() == RaftError.ENOENT.getNumber()) {
```

Comment:
Does this change belong to the ticket itself? It seems to me, this is a separate issue that should be addressed by a separate ticket.

Reply:
I don't see how I can test this separately so it will be just a little commit with 3 lines of code and this will also delay the merging of this ticket, so...

Reply:
I still insist that separate issues should go under separate tickets regardless their size.